db/python/tables/assay.py

```diff
@@ -83,15 +86,30 @@ async def query(
             raise ValueError('Must provide a project if filtering by external_id')
 
         conditions, values = filter_.to_sql(sql_overides)
-        keys = ', '.join(self.COMMON_GET_KEYS)
+        keys = ', '.join(
+            [k for k in self.COMMON_GET_KEYS if include_meta or k != 'a.meta']
+        )
+        meta_slice_keys = (
+            ','.join(
+                [
+                    f'JSON_VALUE(a.meta, \'{m["path"]}\') as _meta_{m["alias"]}'
```

Comment:
definitely a sql injection issue here, would need fixing if we want to do this for real

Reply:
Should also use JSON_EXTRACT rather than JSON_VALUE